experiments/ClimaCore/tc4_atm-lnd-sfc-fluxes/dummy_surface_fluxes.jl

```diff
@@ -94,7 +94,8 @@ function calculate_sfc_fluxes(formulation::DryMonin, parameters, θ_sfc, θ_1, u
         z_0,
         θ_basic,
         z_in,
-        SF.FVScheme(),
+        SF.PointValueScheme(),
+        # TODO Check if statement below is still problematic
```

Comment:
Before we merge this can we see if this #TODO block can be resolved ?

Reply:
we're planning on deleting this test case in the next couple of weeks, so I think it's okay to leave the TODO in this PR